pkg/smi/client_test.go

```diff
@@ -45,7 +45,10 @@ func bootstrapClient() (MeshSpec, *fakeKubeClientSet, error) {
 	smiTrafficSpecClientSet := testTrafficSpecClient.NewSimpleClientset()
 	smiTrafficTargetClientSet := testTrafficTargetClient.NewSimpleClientset()
 	osmPolicyClientSet := osmPolicyClient.NewSimpleClientset()
-	kubernetesClient := k8s.NewKubernetesClient(kubeClient, meshName, stop)
+	kubernetesClient, err := k8s.NewKubernetesClient(kubeClient, meshName, stop)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Error initializing kubernetes controller")
```

Comment:
Instead of `log.Fatal` which calls `os.Exit(1)`, could we use `GinkgoT().Fatal()` or something similar so the failure is registered by the testing framework?

Reply:
Not familiar with `Ginko's fatal`, but seems like we should leverage this across the test code. Would you mind creating an issue to handle this repo wide?

Reply:
What I understand is `GinkgoT()` is just a way to get a handle to the current `*testing.T`, so it's basically the equivalent of `t.Fatal()`. Yes, will open an issue.